src/lib.rs

```diff
@@ -631,27 +631,27 @@ fn optimize_png(
     let output = png.output();
 
     if idat_original_size >= png.idat_data.len() {
-        info!(
+        debug!(
             "    IDAT size = {} bytes ({} bytes decrease)",
             png.idat_data.len(),
             idat_original_size - png.idat_data.len()
         );
     } else {
-        info!(
+        debug!(
             "    IDAT size = {} bytes ({} bytes increase)",
             png.idat_data.len(),
             png.idat_data.len() - idat_original_size
         );
     }
     if file_original_size >= output.len() {
-        info!(
+        debug!(
```

Comment:
It might make sense to leave this and the other one below it in the else block (i.e. both of the ones for "file size") as "info". I think that still accomplishes the goal of this ticket, will reduce the number of log lines from 11 to 1. Moving this to debug would silence everything, which I'm not sure if that is what we want.

Reply:
Yes, it logs only following:
```
info!("Processing from memory")
info!("Image already optimized")
```

Similarly for both `optimize` and `optimize_from_memory`

But yes, since it is size indication, might be ok to leave it as `info`